app/javascript/react/screens/App/Overview/components/Migrations/MigrationsConstants.js

```diff
@@ -0,0 +1,14 @@
+export const MIGRATIONS_NOT_STARTED_SORT_FIELDS = [
+  { id: 'name', title: __('Name'), isNumeric: false },
+  {
+    id: 'configVmLength',
+    title: __('Infrastructure Mapping'),
+    isNumeric: true
+  },
+  { id: 'scheduleTime', title: __('Scheduled Time'), isNumeric: true }
+];
+
+export const MIGRATIONS_COMPLETED_SORT_FIELDS = [
+  { id: 'name', title: __('Name'), isNumeric: false },
+  { id: 'status', title: __('Status'), isNumeric: false }
```

Comment:
Do yah have plans of different status? (TBH I don't)

Reply:
well maybe `mostRecentRequest.status` will do it... 🤔 updated, lemme know if yah see anything different @AparnaKarve

Reply:
Ok, checking...

If it does not work, I can send you a patch to fix this.

Reply:
So let's leave ` { id: 'status', title: __('Status'), isNumeric: false }` as is.

and add these 3 changes -
```diff
diff --git a/app/javascript/react/screens/App/Overview/components/Migrations/Migrations.js b/app/javascript/react/screens/App/Overview/components/Migrations/Migrations.js
index 30e492b..0e28b51 100644
--- a/app/javascript/react/screens/App/Overview/components/Migrations/Migrations.js
+++ b/app/javascript/react/screens/App/Overview/components/Migrations/Migrations.js
@@ -130,7 +130,7 @@ const Migrations = ({
           )}
           {activeFilter === MIGRATIONS_FILTERS.completed && (
             <MigrationsCompletedList
-              finishedTransformationPlans={Immutable.asMutable(finishedTransformationPlans)}
+              finishedTransformationPlans={Immutable.asMutable(finishedTransformationPlans, { deep: true })}
               allRequestsWithTasks={allRequestsWithTasks}
               retryClick={createTransformationPlanRequestClick}
               loading={isCreatingTransformationPlanRequest}
@@ -146,7 +146,7 @@ const Migrations = ({
           )}
           {activeFilter === MIGRATIONS_FILTERS.archived && (
             <MigrationsCompletedList
-              finishedTransformationPlans={archivedTransformationPlans}
+              finishedTransformationPlans={Immutable.asMutable(archivedTransformationPlans, { deep: true })}
               allRequestsWithTasks={allArchivedPlanRequestsWithTasks}
               redirectTo={redirectTo}
               loading={isFetchingArchivedTransformationPlans}

diff --git a/app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js b/app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js
index c12ebec..923e20d 100644
--- a/app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js
+++ b/app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js
@@ -20,6 +20,8 @@ class MigrationsCompletedList extends React.Component {
     const { currentSortType, isSortNumeric, isSortAscending } = this.state;
     const { finishedTransformationPlans } = this.props;
 
+    finishedTransformationPlans.map(plan => plan.status = getMostRecentRequest(plan.miq_requests).status);
+
     return sortFilter(currentSortType, isSortNumeric, isSortAscending, finishedTransformationPlans);
   };
```

Reply:
Can you also add  a `Created` sort option for both Completed and Not Started Plans?

```diff
diff --git a/app/javascript/react/screens/App/Overview/components/Migrations/MigrationsConstants.js b/app/javascript/react/screens/App/Overview/components/Migrations/MigrationsConstants.js
index 11b3624..88c77ab 100644
--- a/app/javascript/react/screens/App/Overview/components/Migrations/MigrationsConstants.js
+++ b/app/javascript/react/screens/App/Overview/components/Migrations/MigrationsConstants.js
@@ -5,10 +5,12 @@ export const MIGRATIONS_NOT_STARTED_SORT_FIELDS = [
     title: __('Infrastructure Mapping'),
     isNumeric: true
   },
-  { id: 'scheduleTime', title: __('Scheduled Time'), isNumeric: true }
+  { id: 'scheduleTime', title: __('Scheduled Time'), isNumeric: true },
+  { id: 'created_at', title: __('Created'), isNumeric: false }
 ];
 
 export const MIGRATIONS_COMPLETED_SORT_FIELDS = [
   { id: 'name', title: __('Name'), isNumeric: false },
-  { id: 'mostRecentRequest.status', title: __('Status'), isNumeric: false }
+  { id: 'status', title: __('Status'), isNumeric: false },
+  { id: 'created_at', title: __('Created'), isNumeric: false }
 ];
```

Reply:
As for the `created`, this diverges from originally specified mock, also, this isn't a value we show on the screen anywhere... would be odd to sort by a hidden value 🤔

Reply:
`created` is the only way to restore the list back to it's original form - which is all the Plans listed in the order they were created.

Reply:
While that may be true, wasn't included in original mock and isn't a field visible to users.  Refreshing the page would also restore original sort.  If we add the field to the list, sure got no problem adding it to the sort (or if @vconzola says do it or else 🔪 ). Otherwise, not a fan of sorting by invisible fields.

Reply:
Ok, fine by me if we don't want to add `created`